src/components/profile/UserProfile.js

```diff
@@ -131,6 +135,15 @@ class UserProfile extends Component {
             this.props.additionalFuncOnSave();
         }
     }
+=======
+                error => {
+                    console.log(error)
+                }
+            )         
+    }
+
+
+>>>>>>> c54d8975517cb6d2c205f225dfe0fc11c71f9ac9
```

Comment:
Leaving branch conflict symbols will lead to unevitable crashes, it won't run! To be removed (you'll have to check which one to keep and discard, or potentially keep/discard both)